content/en/docs/architecture/timeouts.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Job Execution Timeouts"
+date: 2020-12-21T23:27:39-04:00
+draft: false
+---
+
+OpenShift CI/CD platform deploys a [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) instance to manage the lifecycle of jobs. A typical job for Openshift is executed in a Kubernete pod (specified in the job definition and created by Prow) which runs [ci-operator](/docs/architecture/ci-operator/) to assemble an OpenShift release payload, install an ephemeral cluster, run tests against the cluster, and destroy the cluster afterwards. There are timeouts in various phases of a job execution. Some of them are configurable to the developers while others not. In this document, we will explain those timeouts, show the logs if they occur in a job and how to modify their values (if possible).
```

Comment:
This is a good description but I'm going to claim it still goes into way too much detail. The user here cares that:

 - there is a global timeout on the length of a job
 - when that timeout is hit, the job processes receive `SIGINT`

Any and all other details are not necessary.